src/win_job.rs

```diff
@@ -0,0 +1,128 @@
+//! Create a process job so that new processes we create get killed
+//! when we die.
+//!
+//! Adapted from
+//! https://github.com/rust-lang/rust/blob/1.31.1/src/bootstrap/job.rs
```

Comment:
That sounds like a BLATANT VIOLATION OF INTERNATIONAL COPYRIGHT LAW™, yes? =]

```
// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
// file at the top-level directory of this distribution and at
// http://rust-lang.org/COPYRIGHT.
//
// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
// option. This file may not be copied, modified, or distributed
// except according to those terms.
```

IANAL and I might be wrong, but don't you need to include the original copyright/license in your copy, since it's under MIT/Apache 2.0 dual license?